docs/source/whatsnew.rst

```diff
@@ -1,6 +1,37 @@
 What's new in Hyperledger Fabric v2.x
 =====================================
 
+What's New in Hyperledger Fabric v2.4
+-------------------------------------
+
+Fabric Gateway
+^^^^^^^^^^^^^^
+
+The Fabric Gateway is a new service running within peer nodes that enables simplified transaction submission with the following benefits:
+
+* Simplifies client applications and SDKs - Your client application can simply delegate transaction submission to a trusted peer. There is no need for your application to open connections to peer nodes and ordering service nodes from other organizations.
+* Fabric Gateway manages the gathering of transaction endorsements from other organizations and submission to the ordering service on behalf of client applications.
+* Fabric Gateway has intelligence to determine what endorsements are required for a given transaction, even if your solution utilizes a combination of chaincode-level endorsement policies, private data collection endorsement policies, and state-based endorsement policies.
+
+New lightweight gateway SDKs are available for Node, Java, and Go. The SDKs support flexible application patterns:
+
+* You can utilize the high level programming model similar to prior SDK versions, allowing your application to simply call a single SubmitTransaction() function.
+* More advanced applications can leverage the gateway's individual Endorse, Submit, and CommitStatus services for transaction submission, and the Evaluate service for queries.
+* You can entirely delegate transaction endorsement to the gateway, or if needed, specify the endorsing organizations and the gateway will utilize a peer from each organization.
+
+For more information, see the :doc:`gateway` topic.
+
+Peer node unjoin
+^^^^^^^^^^^^^^^^
+
+You can now unjoin a peer from a channel when the channel is no longer needed. All channel resources will be removed from the peer and the peer will no longer process blocks from the channel.
```

Comment:
only when the channel is no longer needed? or at any time? e.g. an org. leaves the channel.

Reply:
I prefer to keep it broad as "is no longer needed". Org leaving a channel is one of the reasons why it may no longer be needed.